README.md

```diff
@@ -7,13 +7,72 @@
 <img alt="Logo Banner" src="https://raw.githubusercontent.com/jellyfin/jellyfin-ux/master/branding/SVG/banner-logo-solid.svg?sanitize=true"/>
 </p>
 
-## Build Process
+## Build With Docker
+
+### Prerequisites
+
+- docker
+- docker-buildx
+- Tizen Studio 4.6+ with IDE (For Samsung certificates, needed once until certificates' expiration)
+
+### Preperations
+
+1. Install prerequisites.
+2. Install Certificate Manager using Tizen Studio Package Manager. See [Installing Required Extensions](https://developer.samsung.com/smarttv/develop/getting-started/setting-up-sdk/installing-tv-sdk.html#Installing-Required-Extensions).
+3. Activate Developer Mode on TV and set Host PC IP to your computer's IP address. See [Enable Developer Mode on the TV](https://developer.samsung.com/smarttv/develop/getting-started/using-sdk/tv-device.html#Connecting-the-TV-and-SDK).
+4. Setup Samsung certificates in Certificate Manager. See [Creating Certificates](https://developer.samsung.com/smarttv/develop/getting-started/setting-up-sdk/creating-certificates.html).
+   > You need to create a Samsung certificate. NOT a Tizen certificate.
```

Comment:
Since some version of Tizen Studio, you can use a Tizen certificate. Maybe only older Tizen versions (2.3/2.4) still require a Samsung or a hybrid certificate.
AFAIK, this is how https://github.com/jeppevinkel/jellyfin-tizen-builds works.

Reply:
I was never able to install using Tizen certificates on my Q60A. Could that be a device/generation specific problem? I am a bit tight on free time this week so can't try right now but will definitely do next week.

Reply:
I at last found the time to spend on this. Really sorry for delaying this much. The situation was completely out of my control.

Now I checked how the certifications work. Looks like for building both Tizen and Samsung certificates work fine. But, since this container also tries to install the app and Tizen certificates are not (if I understood it correctly) signing packages for installation on devices, it's not possible to use them for installation but only emulation. For that, you need a Samsung certificate to sign the package and the device needs to check the validity of the sign from that Samsung account.

If you know how to do the installation with the Tizen certificate I would like to hear that but from my research it looks like it's impossible.